src/style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-image: url('./images/cruel-war-scenes-digital-painting (1).jpg');
+}
```

Comment:
[ OPTIONAL CHANGES ]

The background image style repeats the image and that's a result of some missing styles that you need to add to fix. You could improve your styles with these styles I added below to make your app looks great.

- [ ] Kindly consider the styles below as they will help improve your app's look 👍🏽

```css
body {
  font-family: Arial, Helvetica, sans-serif;
  background-image: linear-gradient(rgb(0,0, 0, 0.4), rgb(0,0,0,0.4)), url('./images/cruel-war-scenes-digital-painting (1).jpg');
  background-repeat: no-repeat;
  background-size: cover;
  background-position: center;
  height: 100vh;
}
```

You can copy all the styles and replace the ones you currently have for the body. I added some styles to your existing body styles.